### PR TITLE
Feature/map centers UI adjustments

### PR DESCRIPTION
**Pull request by DarinHajou**
In this PR, we added the 'Find Me' button to the MapCenter component and made several UI enhancements. Notably, we integrated a scope image to achieve a 'sniper effect,' ensuring the scope remains fixed while the crosshair moves as the user scrolls or interacts with the map

NOTE:

The scope is positioned between approximately 300px to 450px, which should be within acceptable limits for mobile use. However, making sure the scope stays perfectly centered across this range is key for a polished user experience. We might still need to test across different devices to fine-tune the responsiveness.

### CODE REVIEW

File: public/images/icons/map_centers_crosshair.png
Comment (swoocn) [inline public/images/icons/map_centers_crosshair.png:1]: Ooh pretty! 👀

**Comment by swoocn** [comment]
Hi @DarinHajou - were you planning to make some more changes to this PR?

File: public/images/icons/map_centers_crosshair.png
Comment (DarinHajou) [reply public/images/icons/map_centers_crosshair.png:1]: Yes, I need to make some few more changes to this

Comment (DarinHajou) [reply public/images/icons/map_centers_crosshair.png:1]: Ready to merge after review

**Comment by swoocn** [comment]
Hi @DarinHajou - I guess I missed your demo during standup, but is this what we now expect and is PM good with this new layout? [screenshot attached] ->

image

**Review by swoocn — APPROVED** [review]
LGTM @DarinHajou!
Great work 👍
Approved for deployment.

**Pull request merged**